x-pack/agent/pkg/artifact/artifact.go

```diff
@@ -17,7 +17,7 @@ var packageArchMap = map[string]string{
 	"windows-binary-32": "windows-x86.zip",
 	"windows-binary-64": "windows-x86_64.zip",
 	"darwin-binary-32":  "darwin-x86.tar.gz",
```

Comment:
Do we really still have darwin 32 bits?

Reply:
i reverted change of using 64 for 32 bits because when testing with older versions it fails on fetch.